cmd/system-probe/config/adjust_usm.go

```diff
@@ -20,6 +20,7 @@ func adjustUSM(cfg model.Config) {
 	if cfg.GetBool(smNS("enabled")) {
 		applyDefault(cfg, netNS("enable_http_monitoring"), true)
 		applyDefault(cfg, netNS("enable_https_monitoring"), true)
+		applyDefault(cfg, spNS("enable_go_tls_support"), true)
```

Comment:
did you test the change?
a.k.a running USM without configuring go-tls and checking it was enabled?

Reply:
Good catch. I admit I forgot to do so, I should have caught this.